code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -545,3 +545,8 @@
 	icon_state = "bedsheet"
 	user_vars_to_edit = list("name" = "Spooky Ghost", "real_name" = "Spooky Ghost" , "incorporeal_move" = INCORPOREAL_MOVE_BASIC, "appearance_flags" = KEEP_TOGETHER|TILE_BOUND, "alpha" = 150)
 	alternate_worn_layer = ABOVE_BODY_FRONT_LAYER //so the bedsheet goes over everything but fire
+
+/obj/item/clothing/suit/toggle/lawer/black/iaa

```

Comment:
As I told you before, this should be in code/modules/clothing/suits/jobs.dm between the lawyer suits and the mime suspenders.

However, the code itself is correct. Just not the location.